From 58d94e7e887d7c3b3945222d877dcfd0930e9ee8 Mon Sep 17 00:00:00 2001
From: Bjoern <bjoern.riemer@web.de>
Date: Wed, 13 Jan 2016 11:00:29 +0000
Subject: [PATCH 4/4] add --timestamp arg to voctocore

---
 voctocore/voctocore.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/voctocore/voctocore.py b/voctocore/voctocore.py
index 3a4125a..bf9b973 100755
--- a/voctocore/voctocore.py
+++ b/voctocore/voctocore.py
@@ -56,8 +56,9 @@ class Voctocore(object):
 def main():
 	# configure logging
 	docolor = (Args.color == 'always') or (Args.color == 'auto' and sys.stderr.isatty())
+	timestamp = Args.timestamp > 0
 
-	handler = LogHandler(docolor)
+	handler = LogHandler(docolor,timestamp)
 	logging.root.addHandler(handler)
 
 	if Args.verbose >= 2:
-- 
2.5.0

